gent_base.theme

```diff
@@ -373,6 +373,7 @@ function gent_base_preprocess_fieldset(&$variables) {
   }
 
   $variables['has_columns'] = $element['#has_columns'] ?? NULL;
+  $variables['legend']['attributes'][] = $element['#attributes'] ?? NULL;
```

Comment:
Kunnen we hiervoor niet information.attributes in de twig template plaatsen op description field?
Zie gent_base_preprocess_element_information in gent_base.theme